velox/functions/lib/aggregates/tests/utils/AggregationTestBase.cpp

```diff
@@ -550,6 +538,9 @@ bool isTableScanSupported(const TypePtr& type) {
   if (type->kind() == TypeKind::HUGEINT) {
     return false;
   }
+  if (type->kind() == TypeKind::TIMESTAMP) {
```

Comment:
nit: Would you add a comment to explain that this is due xxx issue?

Reply:
Added